german/finger-ins-loch/hoelzernes-Paddel/aufmerksam_machen/aufmerksam_machen.md

```diff
@@ -0,0 +1 @@
+Die Gruppe nimmt Sie zunaechst nicht wahr, sondern tanzt weiter wie in Ekstate. Als Sie in unmittelbare Naehe kommen, entdecken einige Leute Sie und schwimmen zu Ihnen herueber. Das ganze Gruppe dreht sich ploetzlich zu Ihnen um und starrt wie gebannt herueber. Es ist schlagartig leise...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.